api/data.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/ONSdigital/dp-legacy-cache-api/models"
+	"github.com/ONSdigital/log.go/v2/log"
+)
+
+func (api *API) GetDataSets(ctx context.Context) http.HandlerFunc {
```

Comment:
I think its worth writing the tests for these functions once the new `v1/cache-times/` endpoint is setup, otherwise they will just need editing and re writing. Happy either way though.